software-templates/techdocs-template/skeleton/docs/content-templates.md

```diff
@@ -0,0 +1,155 @@
+<!-- Created by Pilar Solares Velasco for DevHub  -->
+# Guide to creating content and templates 
+Use this guide to help you create a strong framework for content creation. Discover valuable resources to lead the development of technical documents in your team's dedicated library in the B.C. government DevHub.
+
+Whether you're enhancing existing content or embarking on new projects, this guide equips you with the essential references and a template to ensure effective communication and compelling documentation.
+
+The content on this page is based on B.C. government resources for web content writing.
+
+## Document template
+Use this document template to create a consistent format for both existing and new pages in your team's documentation library.
+
+### Document title
+Create a title that captures the essence of the documentation.
+
+* The title will contain the only  **H1** heading though the entire the document. 
+
+* Use sentence case for example: 
+
+    * **Correct way**  
+     How to build an application 
+    * **Incorrect way**  
+How To Build An Application  
+How to Build an Application   
+how to build an application 
+
+Sentence case means that in headings and page titles you should only capitalize:
+
+* The first word, if it starts with a number such as '2023 Sysdiq monitor', you capitalize the first word after the number
+
+* [Proper nouns](https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide/capitalization)
+
+* [Abbreviations and acronyms](https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide/abbreviations) if they have already been defined or are better known to the audience than their long-form equivalent, such as PDF or DNA 
+
+### Version history
```

Comment:
I think we can automate this - instead of people needing to manually do it! We have it in our backlog to look at.